src/pages/SubscriptionService.tsx

```diff
@@ -186,7 +209,7 @@ class SubscriptionService extends Component<Props, SubscriptionServiceState> {
       .map(service => this.setState({ service, isValid: true }))
       .mapLeft(() =>
         this.setState({
-          isSaved: false,
+          formState: ServiceFormState.NOT_SAVE,
```

Comment:
In case of a `saved` form, we need to change the state of a form when an input is changed regardless of a `right` or `left` value. Therefore, every time a new value is insert by the user, we need to have a state where the form is `not saved`. So, your question is right, and I will check why I didn't do it before.

Reply:
Suppose you have just saved a form and you got an "ok saved form", then if you switch visible checkbox, you don't remove the message and still the form is in saved mode. So `handleInputChange` is used only by this field, and it's right to handle the new form state also for the right branch.

Reply:
Fix in 8601602